### DISCUSSION TITLE
#18. VATZ biweekly meeting at 2022-09-28 08:00:00 -06:00:00

### DISCUSSION THREAD

**Discussion by xellos00** [dsrvlabs/vatz#251] [General]
### 1. Overall
<img width="1208" alt="image" src="https://user-images.githubusercontent.com/6308023/190290159-de4b58ab-29d0-4f22-be23-4d0eaf373681.png">
<img width="1208" alt="image" src="https://user-images.githubusercontent.com/6308023/190290178-295b483f-e3aa-421a-8c8d-4ed09d9f3dd1.png">

### 2. Statistic Rate
Sprint | Issue fulfillment | progress rate(%)
--: | :--: | :--:
16 | 2/18 | 11
17|  2/19 | 10

---

- Finished
  - #225
  - #195 
    
- Move to Sprint 18
   - #246
   - #242 
   - #238
   - #226  
   - #222 
   - #202 
   - #197 
   - #187 
   - #175 
   - #174 
   - #146 
   - #122 
   - #121 
   - #119 
   - #87
   - #85
     
---

**Comment by xellos00** [comment]
# 동요

- https://github.com/dsrvlabs/vatz/issues/248
   - https://github.com/dsrvlabs/vatz/issues/87 
- https://github.com/dsrvlabs/vatz/issues/240
   - https://github.com/dsrvlabs/vatz/issues/174
   - https://github.com/dsrvlabs/vatz/issues/246

**Comment by rootwarp** [comment]
# 준교

- https://github.com/dsrvlabs/vatz/issues/220

**Comment by kim201212** [comment]
# 형기

- https://github.com/dsrvlabs/vatz/issues/243
   - https://github.com/dsrvlabs/vatz-plugin-cosmoshub/pull/8

**Comment by heejin-github** [comment]
## 희진
- #236 
  - https://github.com/dsrvlabs/vatz-plugin-cosmoshub/pull/11
- #242 
  - #249

**Comment by skonhwang** [comment]
# 승곤
- - https://github.com/dsrvlabs/vatz/issues/244
    - https://github.com/dsrvlabs/vatz-plugin-cosmoshub/pull/12

**Comment by gnongs** [comment]
## 진홍
### Vatz
- https://github.com/dsrvlabs/vatz/issues/226
### Plugin relayer v2
- https://github.com/dsrvlabs/vatz-plugin-relayer-v2/pull/2
- https://github.com/dsrvlabs/vatz-plugin-relayer-v2/pull/3
